src/data/utility/Util.js

```diff
@@ -26,33 +26,39 @@ export const convertObjectIntoURLString = (filters) => {
  * @return {object} Promise reject with statusCode and statusText
  */
```

Comment:
Do we know how this will affect other functions in the webapp that are using the response from the handler?

Reply:
We're adding a new field specifically for the message from the back end so that it won't affect the existing functions.